Resources/Prototypes/ai_factions.yml

```
@@ -248,4 +249,35 @@
+  id: US
+  hostile:
+    - SimpleHostile
+    - Syndicate
+    - Xeno
+    - Zombie
+    - Revolutionary
+    - Dragon
+    - AllHostile
+    - Wizard
+    - Predator
+    - Soviet  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)